search-resources-api/src/main/java/no/unit/nva/search/SearchHandler.java

```diff
@@ -137,6 +138,10 @@ private String extractInstitutionNumberFromRequestedOrganization(URI requestedOr
     }
 
     private String getIndexName(RequestInfo requestInfo) {
-        return attempt(() -> requestInfo.getPathParameter(INDEX)).orElseThrow();
+        return attempt(() -> removeAllSlash(requestInfo.getPath())).orElseThrow();
+    }
+
+    protected static String removeAllSlash(String path) {
+        return path.replaceAll(SLASH, BLANK);
```

Comment:
This works for now but more complex paths will require more processing.

Reply:
Suggestion:
```
    private String getIndexName(RequestInfo requestInfo) {
        return UnixPath.of(requestInfo.getPath()).getFilename();
    }
```
getFilename returns the last part of the path.

Reply:
Agreed